packages/web/src/components/notification/Notifications/ProfilePicture.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback, useEffect, useState } from 'react'
+
+import { push } from 'connected-react-router'
+import { useDispatch } from 'react-redux'
+
+import { SquareSizes } from 'common/models/ImageSizes'
+import { User } from 'common/models/User'
+import ArtistPopover from 'components/artist/ArtistPopover'
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import { useUserProfilePicture } from 'hooks/useUserProfilePicture'
+
+import styles from './ProfilePicture.module.css'
+
+type ProfilePictureProps = {
+  user: User
+  className?: string
+  disablePopover?: boolean
+}
+
+export const ProfilePicture = (props: ProfilePictureProps) => {
+  const { user, className, disablePopover } = props
+  const { user_id, _profile_picture_sizes, handle } = user
+  const [loadImage, setLoadImage] = useState(false)
+  const dispatch = useDispatch()
+  const profilePicture = useUserProfilePicture(
+    user_id,
+    _profile_picture_sizes,
+    SquareSizes.SIZE_150_BY_150,
+    undefined,
+    undefined,
+    loadImage
+  )
+
+  // Loading the images immediately causes lag in the NotificationPanel animation
+  useEffect(() => {
+    if (!loadImage) {
+      const t = setTimeout(() => {
+        setLoadImage(true)
+      }, 500)
```

Comment:
Love this optimizing! Although half a second is pretty long, can it be shorter? also should this be a constant at the top?